### PR TITLE
build(gha): Move `#test-getsentry` parsing out of acceptance workflow

### PR DESCRIPTION
**Pull request by billyvg**
This also changes the commit message parsing to instead use PR body and/or comments.

I considered using labels to trigger the event because 1) it would be more discoverable since you have the list of labels in front of you, 2) have less comment spam (which would notify people subscribed to the issue). However I decided against this because 1) it's less flexible than text (e.g. if we needed commands that takes in arguments) and 2) it triggers an event for each label added.

I changed it from commit messages as they are easy to forget to include and it is kind of a pain to parse them using GH's workflow events.

#test-getsentry

### CODE REVIEW

**Comment by billyvg** [comment]
testing..

File: .github/workflows/sentry-pull-request-bot.yml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+      - name: Dispatch getsentry tests
+        if: steps.org.outputs.result == 'true'
+        uses: actions/github-script@v3
+        with:
+          github-token: ${{ steps.getsentry.outputs.token }}
+          script: |
+            github.actions.createWorkflowDispatch({
+              owner: 'getsentry',
+              repo: 'getsentry',
+              workflow_id: 'acceptance.yml',
```
Comment (joshuarli) [inline .github/workflows/sentry-pull-request-bot.yml:56]: In the future, are you planning on multiple workflow_ids? Or maybe "all" by default. This is definitely a better experience than the manual 1) open getsentry PR, 2) manually bump the correct sentry commit, but perhaps the first step towards a comment-based workflow. Wonder how bors setup compares in difficulty and extensibility.

**Review by joshuarli — APPROVED** [review]
Looks good, I just had a comment for discussion.

File: .github/workflows/sentry-pull-request-bot.yml (hunk 1 above)
Comment (billyvg) [reply .github/workflows/sentry-pull-request-bot.yml:56]: At the moment there are no plans for multiple workflow_ids - I'm thinking we can add more workflows as needed, but currently, javascript changes are generally the biggest cause of build failures.

Yeah I would like to peek into the extensibility of bors, as that is probably a solution we will want.

**Pull request merged**